src/components/page/AddUserPage.tsx

```diff
@@ -0,0 +1,198 @@
+import { useState } from "react";
+import Sidebar from "../common/Sidebar.tsx";
+import Header from "../common/Header.tsx";
+import Footer from "../common/Footer.tsx";
+import apiClient from "../../api/ApiClient.ts";
+import {Link} from "react-router-dom";
+
+const AddUserPage = () => {
+    const [formData, setFormData] = useState({
+        email: "",
```

Comment:
Missing regex for email correct address, I think its wrong place commented tho :D
![2024-12-09_22-43](https://github.com/user-attachments/assets/2c16323c-59df-4f01-8f19-45c362996964)